build.sh

```diff
@@ -122,7 +122,7 @@ else
       --target=$ROOT_DIR/python/ray/pyarrow_files pyarrow==0.12.0.RAY \
       --find-links https://s3-us-west-2.amazonaws.com/arrow-wheels/9357dc130789ee42f8181d8724bee1d5d1509060/index.html
   bazel build //:ray_pkg -c opt
-  # Copy files and do not overwrite ownership
+  # Copy files and keep them writeable.
```

Comment:
If it's a hack, can you add a note explaining that it's a hack and why we're doing it?

Reply:
I added the comment.